app.rb

```diff
@@ -46,12 +46,8 @@ class AbodenB < Sinatra::Base
   end
 
   post "/listings/new" do
```

Comment:
As a non-user I can type the URL and it let me in, add a condition.

Reply:
fixed by Will & Javi